CHANGELOG.md

```diff
@@ -38,6 +38,7 @@
 
 * Added the `MapboxSpeechSynthesizer(remoteSpeechSynthesizer:)` initializer for creating a `MapboxSpeechSynthesizer` object that uses a custom `SpeechSynthesizer` instance. ([#3747](https://github.com/mapbox/mapbox-navigation-ios/pull/3747))
 * MapboxNavigationNative now sends messages to the Unified Logging subsystem `com.mapbox` for easier filtering. ([#3765](https://github.com/mapbox/mapbox-navigation-ios/pull/3765))
+* Fixed a bug which caused `RouteOptions.profileIdentifier` to be ignored in active navigation. ([#3796](https://github.com/mapbox/mapbox-navigation-ios/pull/3796))
```

Comment:
could you add a few words about the behavior this pr fixes? like:
```suggestion
* Fixed a bug which caused `RouteOptions.profileIdentifier` to be ignored in active navigation and elevated network usage. ([#3796](https://github.com/mapbox/mapbox-navigation-ios/pull/3796))
```

Reply:
Yeah, I'll mention that it may have caused it, but it was only a side effect.

Also, this bug was introduced in `2.4.0-beta.2` and fixed right away in `2.4.0-beta.3`, so I don't want the whole changelog for version 2.4.0 to look like we had a big issue.